config/settings.yml

```diff
@@ -109,13 +109,12 @@ features:
     has_current_cycle_started?: true
     # During rollover providers should be able to edit current & next recruitment cycle courses
     can_edit_current_and_next_cycles: false
-  # This is a temporary feature flag to allow time to link the gov style feedback form responses to the support console.
-  # N.B. gov_style_feedback_form_enabled is set to false as a default and true in review and test.
-  # To clean up: you can delete the feature flag, remove the helper method here: app/helpers/find/feedback_form_helper.rb, change the url and new_tab: to false in layouts/find, remove the feature flag from the routes.rb file if it remains.
-  gov_style_feedback_form_enabled: false
 
 cookies:
   session:
     name: _teacher_training_courses_session
 
 STATE_CHANGE_SLACK_URL: replace_me
+
+find_service:
+  feedback_url: "./feedback/new"
```

Comment:
We can remove this line and use the routes. `new_find_feedback_path`.

Happy to pair if you like :)